### DISCUSSION TITLE
[Feature Request] Queryable Encryption or CSFLE

### DISCUSSION THREAD

**Discussion by mamtaRaiTech** [jenssegers/laravel-mongodb#3229] [General]
Feature Request: Automatic Encryption for MongoDB Connection
Is your feature request related to a problem?
No, but implementing this feature would ensure a smoother transition when using MongoDB with Node.js.

Describe the solution you'd like
I would like to explore the possibility of enabling automatic encryption for MongoDB connections. This feature would provide seamless data security without requiring significant manual configuration or code changes.

Describe alternatives you've considered
Currently, the alternative involves manually configuring encryption in Node.js applications, which can be error-prone and time-consuming. An integrated solution within the connection setup would simplify this process.

**Comment by bisht2050** [comment]
This looks like a spam/bot account. 
 
For anyone genuinely interested in Queryable Encryption with Laravel - It is tracked with [PHPORM-136](https://jira.mongodb.org/browse/PHPORM-136)

**Reply by alcaeus** [reply]
Ah, yes, this is a carbon copy of #2699.
